pootle/apps/pootle_translationproject/models.py

```diff
@@ -191,11 +191,15 @@ def fullname(self):
 
     @property
     def abs_real_path(self):
-        return absolute_real_path(self.real_path)
+        if self.real_path is not None:
+            return absolute_real_path(self.real_path)
```

Comment:
no, i want it to return none if no project_tree is set up, that way there can be no possibility of anything on disk interfering with Stores